src/platform/plugins/shared/inspector/public/ui/inspector_view_chooser.tsx

```diff
@@ -69,57 +63,44 @@ export class InspectorViewChooser extends Component<Props, State> {
     );
   };
 
-  renderViewButton() {
-    return (
-      <EuiButtonEmpty
-        size="s"
-        iconType="arrowDown"
-        iconSide="right"
-        onClick={this.toggleSelector}
-        data-test-subj="inspectorViewChooser"
-      >
-        <FormattedMessage
-          id="inspector.view"
-          defaultMessage="View: {viewName}"
-          values={{ viewName: this.props.selectedView.title }}
-        />
-      </EuiButtonEmpty>
-    );
-  }
-
   renderSingleView() {
     return (
-      <EuiToolTip position="bottom" content={this.props.selectedView.help}>
-        <FormattedMessage
-          id="inspector.view"
-          defaultMessage="View: {viewName}"
-          values={{ viewName: this.props.selectedView.title }}
-        />
-      </EuiToolTip>
+      <EuiTabs size="s">
+        <EuiToolTip position="bottom" content={this.props.selectedView.help}>
+          <EuiTab
+            isSelected={true}
+            data-test-subj={`inspectorViewChooser${this.props.selectedView.title}`}
+          >
+            <FormattedMessage
+              id="inspector.view"
+              defaultMessage="View: {viewName}"
```

Comment:
If design wants to keep the single tag, how about just using `defaultMessage={viewName}`?